### PR TITLE
feat: add new gauges on arbitrum eth and bsc

### PR DESCRIPTION
**Pull request by ChefBingbong**
---

### Focus of the PR:
This PR focuses on adding new ERC20 tokens and updating farm configurations.

### Detailed summary:
- Added a new ERC20 token `wrose` with its details.
- Added a new ERC20 token `swETH` with its details.
- Updated the list of tokens in `packages/farms/constants/common/index.ts`.
- Added new ERC20 tokens `usdplus`, `usdce`, `usdtplus`, and `ethplus` with their details.
- Updated the farm configurations in `packages/farms/constants/arb.ts` and `packages/farms/constants/bsc.ts`.
- Added new gauge configurations in `packages/gauges/src/constants/config/prod.ts`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/gauges/src/constants/config/prod.ts [hunk 1]
```diff
@@ -203,7 +203,7 @@ export const CONFIG_PROD: GaugeConfig[] = [
     type: GaugeType.V3,
     token0Address: bscTokens.ankrETH.address,
     token1Address: bscTokens.eth.address,
-    feeTier: FeeAmount.LOW,
+    feeTier: FeeAmount.LOWEST,
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:206]: Should maintain `LOW`

File: packages/tokens/src/56.ts [hunk 2]
```diff
@@ -2892,4 +2892,5 @@ export const bscTokens = {
     'INSPECT',
     'https://www.inspect.xyz/',
   ),
+  wrose: new ERC20Token(ChainId.BSC, '0xF00600eBC7633462BC4F9C61eA2cE99F5AAEBd4a', 18, 'WROSE', 'Wrapped ROSE', ''),
```
Comment (ChefMomota) [inline packages/tokens/src/56.ts:2895]: Fill the url

File: packages/tokens/src/42161.ts [hunk 3]
```diff
@@ -183,4 +183,22 @@ export const arbitrumTokens = {
...
+  usdce: new ERC20Token(
+    ChainId.ARBITRUM_ONE,
+    '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8',
+    6,
+    'USD Coin (Arb1)',
+    'USDC.e',
+    'https://www.centre.io/',
+  ),
+  usdtplus: new ERC20Token(ChainId.ARBITRUM_ONE, '0xb1084db8D3C05CEbd5FA9335dF95EE4b8a0edc30', 6, 'USDT+', 'USDT+', ''),
+  ethlus: new ERC20Token(ChainId.ARBITRUM_ONE, '0xD4939D69B31fbE981ed6904A3aF43Ee1dc777Aab', 18, 'ETH+', 'ETH+', ''),
```
Comment (ChefMomota) [inline packages/tokens/src/42161.ts:203]: Fill the url

Comment (ChefMomota) [inline packages/tokens/src/42161.ts:203]: ethplus or ethlus

File: packages/tokens/src/42161.ts [hunk 4]
```diff
@@ -183,4 +183,22 @@ export const arbitrumTokens = {
...
+    'USD+',
+    'USD+',
+    'https://overnight.fi/',
+  ),
+  usdce: new ERC20Token(
+    ChainId.ARBITRUM_ONE,
+    '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8',
+    6,
+    'USD Coin (Arb1)',
+    'USDC.e',
```
Comment (ChefMomota) [inline packages/tokens/src/42161.ts:199]: Change the sort. `USDC.e` must above  `USD Coin (Arb1)`

File: packages/gauges/src/constants/config/prod.ts [hunk 5]
```diff
@@ -2207,6 +2198,94 @@ export const CONFIG_PROD: GaugeConfig[] = [
     type: GaugeType.V3,
     token0Address: polygonZkEvmTokens.weth.address,
     token1Address: polygonZkEvmTokens.wbtc.address,
+    feeTier: FeeAmount.MEDIUM,
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:2201]: LOW

File: packages/gauges/src/constants/config/prod.ts [hunk 6]
```diff
@@ -2207,6 +2198,94 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    gid: 222,
+    pairName: 'HAY-USDT',
+    address: '0xB2Aa63f363196caba3154D4187949283F085a488',
+    chainId: ChainId.BSC,
+    type: GaugeType.StableSwap,
+    tokenAddresses: [bscTokens.hay.address, bscTokens.usdt.address],
+  },
+  {
+    gid: 223,
+    pairName: 'CGPT-USDT',
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:2223]: USDT-CGPT

File: packages/gauges/src/constants/config/prod.ts [hunk 7]
```diff
@@ -2207,6 +2198,94 @@ export const CONFIG_PROD: GaugeConfig[] = [
     type: GaugeType.V3,
     token0Address: polygonZkEvmTokens.weth.address,
     token1Address: polygonZkEvmTokens.wbtc.address,
+    feeTier: FeeAmount.MEDIUM,
+  },
+  {
+    gid: 221,
+    pairName: 'swETH-ETH',
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:2205]: ETH-swETH

File: packages/gauges/src/constants/config/prod.ts [hunk 8]
```diff
@@ -2207,6 +2198,94 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    feeTier: FeeAmount.MEDIUM,
+  },
+  {
+    gid: 226,
+    pairName: 'USD+-USDT+',
+    address: '0xb9c2d906f94b27bC403Ab76B611D2C4490c2ae3F',
+    chainId: ChainId.ARBITRUM_ONE,
+    type: GaugeType.V3,
+    token0Address: arbitrumTokens.usdplus.address,
+    token1Address: arbitrumTokens.usdtplus.address,
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:2258]: token0 is usdt not usd

File: packages/gauges/src/constants/config/prod.ts [hunk 9]
```diff
@@ -2207,6 +2198,94 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    address: '0xf0B860d338E8B5199606322653B83A166d96E417',
+    chainId: ChainId.ARBITRUM_ONE,
+    type: GaugeType.V3,
+    token0Address: arbitrumTokens.xai.address,
+    token1Address: arbitrumTokens.weth.address,
+    feeTier: FeeAmount.MEDIUM,
+  },
+  {
+    gid: 226,
+    pairName: 'USD+-USDT+',
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:2253]: USDT+-USD+

File: packages/gauges/src/constants/config/prod.ts [hunk 10]
```diff
@@ -2207,6 +2198,94 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    address: '0x06c75011479E47280e8B7E72E9e0315C8b3A634d',
+    chainId: ChainId.ARBITRUM_ONE,
+    type: GaugeType.V3,
+    token0Address: arbitrumTokens.ethplus.address,
+    token1Address: arbitrumTokens.usdplus.address,
+    feeTier: FeeAmount.MEDIUM,
+  },
+  {
+    gid: 228,
+    pairName: 'USDC.e-USD+',
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:2273]: USD+-USDC.e

File: packages/gauges/src/constants/config/prod.ts [hunk 11]
```diff
@@ -2207,6 +2198,94 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+  },
+  {
+    gid: 229,
+    pairName: 'INSP-BNB',
+    address: '0x06aC8EE32BCdcE6bF2eA82D9Bfb932a84D45BFcb',
+    chainId: ChainId.BSC,
+    type: GaugeType.V3,
+    token0Address: bscTokens.insp.address,
+    token1Address: bscTokens.wbnb.address,
+    feeTier: FeeAmount.LOWEST,
```
Comment (ChefMomota) [inline packages/gauges/src/constants/config/prod.ts:2289]: HIGH

File: packages/gauges/src/constants/config/prod.ts [hunk 12]
```diff
@@ -1212,7 +1203,7 @@ export const CONFIG_PROD: GaugeConfig[] = [
     type: GaugeType.V3,
     token0Address: ethereumTokens.rETH.address,
     token1Address: ethereumTokens.weth.address,
-    feeTier: FeeAmount.LOW,
+    feeTier: FeeAmount.HIGH,
```
[the remark this answers is not in the document]
Comment (ChefBingbong) [reply packages/gauges/src/constants/config/prod.ts:1206]: some guages had to be changes see the post. fees in some had to be edited

File: packages/gauges/src/constants/config/prod.ts (hunk 1 above)
Comment (ChefBingbong) [reply packages/gauges/src/constants/config/prod.ts:206]: if you check the gauges config from chat it sid t change the fee on this

File: packages/gauges/src/constants/config/prod.ts (hunk 12 above)
Comment (chefjackson) [inline packages/gauges/src/constants/config/prod.ts:1206]: this is the feeTier of the pool. Should follow the pool

File: packages/gauges/src/constants/config/prod.ts (hunk 1 above)
Comment (chefjackson) [reply packages/gauges/src/constants/config/prod.ts:206]: this is the fee tier of the pool. Should be fixed

File: pnpm-lock.yaml
Comment (chefjackson) [inline pnpm-lock.yaml:1]: why are we updating the lockfile?

Comment (ChefBingbong) [reply pnpm-lock.yaml:1]: i ran pnpm build backages

File: packages/farms/constants/common/index.ts [hunk 13]
```diff
@@ -42,7 +42,14 @@ export const priceHelperTokens = {
...
   [ChainId.ARBITRUM_ONE]: {
     chain: 'arbitrum',
-    list: [arbitrumTokens.weth, arbitrumTokens.usdc, arbitrumTokens.usdt, arbitrumTokens.arb],
+    list: [
+      arbitrumTokens.weth,
+      arbitrumTokens.usdc,
+      arbitrumTokens.usdt,
+      arbitrumTokens.arb,
+      arbitrumTokens.usdplus,
+      arbitrumTokens.usdtplus,
```
Comment (chefjackson) [inline packages/farms/constants/common/index.ts:51]: I think we only need usdplus. Let's try remove `usdtplus`

File: packages/gauges/src/constants/config/prod.ts [hunk 14]
```diff
@@ -2209,4 +2180,172 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    address: '0x1E8D7AB96323953faC85ab687f5D38b45344e62d',
+    chainId: ChainId.BSC,
+    type: GaugeType.V3,
+    token0Address: bscTokens.usdt.address,
+    token1Address: bscTokens.cgpt.address,
+    feeTier: FeeAmount.LOW,
+  },
+  {
+    gid: 224,
+    pairName: 'USDT-WROSE',
```
Comment (ChefJerry) [inline packages/gauges/src/constants/config/prod.ts:2213]:
```suggestion
    pairName: 'USDT-wROSE',
```

File: packages/gauges/src/constants/config/prod.ts [hunk 15]
```diff
@@ -2209,4 +2180,172 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    address: '0x06aC8EE32BCdcE6bF2eA82D9Bfb932a84D45BFcb',
+    chainId: ChainId.BSC,
+    type: GaugeType.V3,
+    token0Address: bscTokens.insp.address,
+    token1Address: bscTokens.wbnb.address,
+    feeTier: FeeAmount.HIGH,
+  },
+  {
+    gid: 226,
+    pairName: 'BTCB-FDUSD',
```
Comment (ChefJerry) [inline packages/gauges/src/constants/config/prod.ts:2233]:
```suggestion
    pairName: 'FDUSD-BTC',
```

File: packages/gauges/src/constants/config/prod.ts [hunk 16]
```diff
@@ -2209,4 +2180,172 @@ export const CONFIG_PROD: GaugeConfig[] = [
...
+    address: '0x8b626Acfb32CDad0d2F3b493Eb9928BbA1BbBcCa',
+    chainId: ChainId.BSC,
+    type: GaugeType.V3,
+    token0Address: bscTokens.ckp.address,
+    token1Address: bscTokens.wbnb.address,
+    feeTier: FeeAmount.MEDIUM,
+  },
+  {
+    gid: 237,
+    pairName: 'CKP-BNB',
```
Comment (ChefJerry) [inline packages/gauges/src/constants/config/prod.ts:2343]:
```suggestion
    pairName: 'IRL-USDT',
```

**Review by ChefJerry — APPROVED** [review]

**Pull request merged**